app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotWearingMaskError,
+    NotVaccinatedError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if vaccine := visitor.get("vaccine"):
+            if datetime.date.today() > vaccine.get("expiration_date"):
+                raise OutdatedVaccineError
```

Comment:
According to the checklist, you should use descriptive error messages. Here, you're raising `OutdatedVaccineError` without any message. It would be better to provide a message that describes what went wrong. For example, `raise OutdatedVaccineError('Your vaccine is outdated.')`.